lib/msal-core/src/telemetry/CacheEvent.ts

```diff
@@ -0,0 +1,30 @@
+import TelemetryEvent from "./TelemetryEvent";
+import { EVENT_NAME_PREFIX } from "./TelemetryConstants";
+
+export const CACHE_EVENT_TYPES = {
+    TokenCacheLookup: `${EVENT_NAME_PREFIX}token_cache_lookup`,
+    TokenCacheWrite: `${EVENT_NAME_PREFIX}token_cache_write`,
+    TokenCacheBeforeAccess: `${EVENT_NAME_PREFIX}token_cache_before_access`,
+    TokenCacheAfterAccess: `${EVENT_NAME_PREFIX}token_cache_after_access`,
+    TokenCacheBeforeWrite: `${EVENT_NAME_PREFIX}token_cache_before_write`,
+    TokenCacheDelete: `${EVENT_NAME_PREFIX}token_cache_delete`,
```

Comment:
We are appending EVENT_NAME_PREFIX at many places to constant strings. Is there an easier way to do that?

Reply:
yes! utilitizing

Reply:
added util fixed everywhere, good suggestion